Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v3.yml

```diff
@@ -616,12 +754,6 @@ inputs:
     This input determines whether Palo Alto Networks Panorama or Firewall Static Address Groups are used.
     Specify the Static Address Group name for IPs list handling.
   playbookInputQuery:
-- key: URLListName
-  value:
-    simple: Demisto Remediation - URL EDL
-  required: false
-  description: URL list from the instance context with which to override the remote file.
-  playbookInputQuery:
 - key: CustomURLCategory
   value:
     simple: Demisto Remediation - Malicious URLs
```

Comment:
`Block URL - Generic v2` is using "XSOAR Remediation - Malicious URLs". Please change the input for `Block Indicators - Generic v3` to be "XSOAR Remediation - Malicious URLs" and then in `Block URL - Generic v2` use the input. This way we fix it to use the input while preserving backward compatibility